.travis.yml

```diff
@@ -3,21 +3,12 @@ node_js: node
 before_install:
 - export DISPLAY=:99.0
 - sh -e /etc/init.d/xvfb start
-- npm i -g greenkeeper-lockfile@1
```

Comment:
I can bring this back if desired.  Removed it accidentally, b/c I started with a fresh `.travis.yml` when I ran `travis setup npm`.

Reply:
No need since we nixed the `package-lock.json`